framework/math/quadratures/angular/sldfe_sq_quadrature.cc

```diff
@@ -1,12 +1,12 @@
-// SPDX-FileCopyrightText: 2024 The OpenSn Authors <https://open-sn.github.io/opensn/>
+// SPDX-FileCopyrightText: 2024 The OpenSn Authors <https://open-sn.github.io/opensn/> 
```

Comment:
Remove the trailing space. 

You may want to consider setting your text editor/IDE to remove trailing spaces on save to prevent these things.
If your editor/IDE supports [editor config](https://editorconfig.org/) (there are plugins for that), you may want to use that, then the editor will pick up the settings from our supplied `.editorconfig` and will make your life easier.